app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,85 @@
+class AttendancesController < ApplicationController
+  before_action :set_attendance, only: [:show, :edit, :update, :destroy]
+
+  # GET /attendances
+  # GET /attendances.json
+  def index
+    # @select_options = {
+    #   semana: :day_of_week,
+    #   mes: :day_of_month,
+    #   año: :month_of_year
+    # }
+    # @time_period = params[:time_period] || :semana
+
+    @start_date = params[:start_date]
+    if @start_date.nil?
+      @start_date = Time.now.beginning_of_week
+    else
+      @start_date = Time.strptime(@start_date, "%m/%d/%Y")
+    end
+
+    @end_date = params[:end_date]
+    if @end_date.nil?
+      @end_date = Time.now.end_of_week
+    else
+      @end_date = Time.strptime(@end_date, "%m/%d/%Y")
+    end
+
+    @attendances = current_user.attendances
+    @attendances = @attendances.group_by_period(:day_of_week, :date, range: @start_date..@end_date, format: "%a").count
+    puts @attendances
+    @attendances_count = @attendances.sum{ |k, v| v }
```

Comment:
This method is way too long, let's refactor what we can out to different methods please!

Reply:
Just created method for `@start_date` and `@end_date` initialization